rust-atuin.spec

```diff
@@ -0,0 +1,80 @@
+# Generated by rust2rpm 25
+
+# prevent library files from being installed
+%global __cargo_is_lib() 0
+
+%global crate atuin
+
+Name:           rust-atuin
+Version:        18.0.2
+Release:        %autorelease
+Summary:        Magical shell history
+
+License:        MIT
+URL:            https://crates.io/crates/atuin
+Source:         %{crates_source}
+Source:         atuin-%{version}-vendor.tar.xz
+
+BuildRequires:  cargo-rpm-macros >= 25
+
+%global _description %{expand:
+Atuin replaces your existing shell history with a SQLite database, and records
+additional context for your commands. Additionally, it provides optional and
+fully encrypted synchronisation of your history between machines, via an Atuin
+server.}
+
+%description %{_description}
+
+%package     -n %{crate}
```

Comment:
I might be wrong, but looking at the [builds in COPR](https://copr.fedorainfracloud.org/coprs/sramanujam/atuin/build/7127526/), it looks like the package is being named as `rust-atuin` and not `atuin` like I would expect from this config. 

Is that expected?

Reply:
[Here](https://copr.fedorainfracloud.org/coprs/packit/LecrisUT-atuin-rpmspec-1/build/7113489/) you go. This was a previous run without the `check`. It takes the name of the crate, in this case it is `atuin`. Note that when we decouple the `vendor`, we will use the git repo + more explicit naming.